pg_shard.c

```diff
@@ -1045,6 +1653,23 @@ CreateTemporaryTableLikeStmt(Oid sourceRelationId)
 
 
 /*
+ * CreateUniqueTableName creates a unique table name on every invocation. It takes
+ * no parameters and returns a StringInfo.
+ */
+static StringInfo
+CreateUniqueTableName(void)
```

Comment:
I don't remember this function being around earlier. What is it necessary for now (and can we just use a file static variable instead of this function-scoped one?)